tmt/steps/execute/internal.py

```diff
@@ -648,7 +648,11 @@ def pull_from_guest() -> None:
         # and it's hard to pick the main one, who knows what custom results might
         # cover, so let's make sure every single result can lead to check results
         # related to its lifetime.
+
+        submitted_files = invocation.submitted_files
         for result in invocation.results:
+            if result.name == invocation.test.name:
+                result.log.extend(submitted_files)
```

Comment:
You can use `invocation.submitted_files` here, directly, without a temporary variable.

Reply:
seems resolved to me.